homeassistant/components/venstar/sensor.py

```diff
@@ -210,3 +229,16 @@ def native_unit_of_measurement(self) -> str | None:
     value_fn=lambda coordinator, sensor_name: coordinator.runtimes[-1][sensor_name],
     name_fn=lambda coordinator, sensor_name: f"{coordinator.client.name} {RUNTIME_ATTRIBUTES[sensor_name]} Runtime",
 )
+
+INFO_ENTITIES: tuple[VenstarSensorEntityDescription, ...] = (
+    VenstarSensorEntityDescription(
+        key="schedulepart",
+        device_class=SensorDeviceClass.ENUM,
+        options=list(SCHEDULE_PARTS.values()),
+        uom_fn=lambda _: None,
```

Comment:
This could be the default in the `VenstarSensorEntityDescription`.

Reply:
This would be neat, but the way the mixin is currently implemented means that there will be inter-mixed default and non-default arguments, which Python isn't happy with, so this would require more rearchitecting than I think is the goal here:
```diff
-     uom_fn: Callable[[Any], str | None]
+    uom_fn: Callable[[Any], str | None] = lambda _: None
```

`TypeError: non-default argument 'key' follows default argument`

Reply:
We would move the attribute to the `VenstarSensorEntityDescription`.

This was a side note so we don't need to do it now.